solution.md

```diff
@@ -0,0 +1,17 @@
+* cd mystery
+* ls
+* cat crimescene
+* //too much crap
+* //grep 'CLUE' crimescene
+* //AAA, DELTA SkyMiles library museum of bash history,
+* //male, 6feet tall
+//Mike Bostock	M	45	Senders Court, line 287
+* cd interviews
+* grep -r -e '6'
```

Comment:
This will only bring up the first person who is over 6ft, so you might have missed some other possibilities.